### PR TITLE
[Non-Modular] Rebalances previous SM anom PR

### PR DESCRIPTION
**Pull request by NotRanged**
## About The Pull Request

I previously re-enabled the SM spawning core-dropping anomalies but gave it a low chance to do so. This PR rebalances the base drop rate.

## Why It's Good For The Game
I've seen people run high EER supermatters a few times now to spawn the anoms, and did it myself a few times too, and the drop rate was *way* too low. Especially compared to the danger involved, /tg/ anomalies are pretty lethal turns out. At the current numbers and with some testing, it seemed a lot more fair to do.

## Changelog
:cl:
balance: Increased the base chance for a supermatter to spawn anomaly-core dropping anomalies to 25%. 
/:cl:

### CODE REVIEW

File: code/modules/power/supermatter/supermatter.dm [hunk 1]
```diff
@@ -1107,7 +1107,7 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 	var/drops_core = FALSE
 	// Every 334 EER above POWER_PENALTY_THRESHOLD adds 10% chance for the anomalies to leave behind a core when neutralized
 	var/chance_bonus = clamp((10 * (power - POWER_PENALTY_THRESHOLD) / 334), 0, 95)
-	if(prob(5 + round(chance_bonus))) // 5% base chance to drop a core
+	if(prob(25 + round(chance_bonus))) // 25% base chance to drop a core
```
Comment (GuppyLaxx) [inline code/modules/power/supermatter/supermatter.dm:1110]: Slap a fat fucking `- SKYRAT EDIT` or something on there and you're golden

Comment (NotRanged) [reply code/modules/power/supermatter/supermatter.dm:1110]: This already has edit comments

**Comment by NotRanged** [comment]
Edit comments are already present at line 1106 and 1122

**Review by Gandalf2k15 — APPROVED** [review]

**Comment by Sethnov** [comment]
would it be bad if we add export value to the cores?

**Pull request merged**